include/openenclave/internal/sgxsign.h

```diff
@@ -36,6 +38,8 @@ oe_result_t oe_sgx_sign_enclave(
     uint16_t security_version,
     const uint8_t* pem_data,
     size_t pem_size,
+    const uint8_t* family_id,
+    const uint8_t* ext_product_id,
```

Comment:
```suggestion
    const uint8_t* extended_product_id,
```

Reply:
Updated per comment

Reply:
Can you do a global find and replace for this? In this iteration, the only fixed value I see is on line 75 and there are many instances in the EDL, comments and functions that remain `ext_product_id`

Reply:
Thanks Replaced in #3700